tests/integration/populate_notary.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Does this actually need a separate script? Just for reading notary_init && notary_addhash $X sounds similarly reasonable to populate_notary $X